src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1282,9 +1294,9 @@ static void addBsonFilter
   }
   else
   {
-    std::string details = std::string("unknown query operator: ") + opr;
+    std::string details = std::string("unknown query operator: '") + opr + "'";
```

Comment:
As far as I know, it is not used. Might be used in the future though.
Change for /  ?